tests/test_pipeline_config.py

```diff
@@ -20,103 +20,46 @@ def setUp(self):
         with codecs.open(test_config_folder+'/config.cfg',mode='w',encoding='utf-8') as f:
             f.write(
 '''
-[local_pipeline_setting]
-POS = true
-LEMMA = true
-NER_CONLL = false
-NER_ONTONOTES = false
-QUANTITIES = false
-SHALLOW_PARSE = false
-SRL_VERB = false
-DEPENDENCY_STANFORD = false
-DEPENDENCY = false
-PARSE_STANFORD = false
-SRL_PREP = false
-
 [remote_pipeline_setting]
-api = http://austen.cs.illinois.edu:8080/annotate
+api = http://austen.cs.illinois.edu:7080
```

Comment:
It doesn't matter, tests for pipeline_config only check if the module loads config correctly